### ISSUE TITLE
Tab curve is cut off when selecting a tab from the overflow dropdown menu.

### ISSUE THREAD

**Issue by jasonbassford** [wilfredwee/photon-australis#72]
I'm not sure if there is a technical solution for this or not, and it's only a minor annoyance, but I wanted to mention it.

To reproduce:

1. Open enough tabs that you get tab overflow, and that there are so many that there are multiple "rows." (In other words, if you only see 10 tabs at once, open 25 tabs.)
2. From the tab menu, switch to a tab that's at the opposite end of the list as the tab that you're currently viewing.

When the tab bar redraws itself, the "curve" at the bottom of the left or right of the now-active tab is not shown.

If I've jumped to the first tab in the list, I have to click the left tab-overflow scroll arrow in order to reposition it _just_ enough more that I can now see the left-hand curve of that tab. (Or, if I've jumped to the last tab in the list, I have to click the right tab-overflow scroll arrow in order to see the right-hand curve of the last tab.)

In short, the position of the overflowed content doesn't move _completely_ to the left or right when selecting the tab from the dropdown list. It excludes the extra curve that's been styled to the tab's left or right.

I did some quick Google searching for CSS styling that could be applied to the tab overflow bar, but was unable to find anything that seemed pertinent.

**Comment by wilfredwee** [comment]
I've looked through the code and trying various things. Problem is that the tab margin for the pinned tabs are programmatically set by Firefox during runtime. There's no way to fix this without introducing custom JS script.

Marking as an enchancement fix.
